tests/TestCase/Log/Engine/SyslogLogTest.php

```diff
@@ -110,6 +110,18 @@ public function testDeprecatedFormat()
         });
     }
 
+    /**
+     * Test deprecated format option
+     *
+     * @return void
+     */
+    public function testDeprecatedFormatMessage()
+    {
+        $this->expectDeprecation();
+        $this->expectDeprecationMessage('`format` option is now deprecated in favour of custom formatters');
```

Comment:
```suggestion
        $this->expectDeprecationMessage('`format` option is now deprecated in favor of custom formatters');
```
We use AE usually in coding (colour vs color etc, behavior, ...)